client/lclusterc.go

```diff
@@ -0,0 +1,316 @@
+/*
+ * File: client/main.go
+ *
+ * Description: Contains the lcluster client code.
+ *
+ * TODO: removejob is broken, others funcs are unimplemented
+ */
+
+package main
```

Comment:
Added additional content to the README.md file.

https://github.com/rbisewski/lclusterd/commit/3a98e2d63260157f1b8690e884d4ae4e3f9149b7